### PR TITLE
Alerts/Monitors - grammar, formatting cleanup

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Alerts/Monitors - grammar, formatting cleanup

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/alerts/monitors/alert-grouping.md [hunk 1]
```diff
@@ -15,111 +15,111 @@ You could group by `_collector` field, for example, and one alert would be gener
...
-5. Configure the rest of your Alert Condition per standard procedure. Refer to [Monitors ](/docs/alerts/monitors)for more details.
+3. Select **Metrics** as the type of monitor.
+4. Enter your metrics query, then select your desired alert grouping option.
+   * **One alert per monitor**. If you only want to receive a single alert for the entire monitor.
+   * **One alert per time series**. To receive a single alert for each time-series that is present in the metric query
+   * **One alert per [group]**. allows you to receive one notification per each unique value of the grouping field(s). You can pick more than one field for the grouping condition. In the example below, user will receive one notification when CPU utilization is higher than the threshold for each unique AWS namespace within an account.<br/><img src={useBaseUrl('img/monitors/setup-metrics.png')} alt="setup-metrics.png" />
```
Comment (jpipkin1) [inline docs/alerts/monitors/alert-grouping.md:29]:
```suggestion
   * **One alert per [group]**. Allows you to receive one notification per each unique value of the grouping field(s). You can pick more than one field for the grouping condition. In the example below, user will receive one notification when CPU utilization is higher than the threshold for each unique AWS namespace within an account.<br/><img src={useBaseUrl('img/monitors/setup-metrics.png')} alt="setup-metrics.png" />
```

File: docs/alerts/monitors/create-monitor.md [hunk 2]
```diff
@@ -17,31 +17,31 @@ This topic shows you how to create a monitor.
 </details>
 
 
-## Open the New Monitor window
+## Step 1. Open the New Monitor window
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:20]: @kimsauce, I don't think you should make this "Step 1". Notice how the sections of the New Monitor dialog are numbered 1-5. To reduce confusion for readers, I think we need to align the numbers of the steps with the numbers of the dialog sections. What do you think?

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@kimsauce, great cleanup! I have only a couple of suggestions.

File: docs/alerts/monitors/create-monitor.md (hunk 2 above)
Comment (kimsauce) [reply docs/alerts/monitors/create-monitor.md:20]: Oops, you're right! I was wondering why I hadn't labeled that "Step 1" in the first place. Thanks for pointing that out.

**Review by jpipkin1 — APPROVED** [review]

**Comment by jpipkin1** [comment]
@kimsauce, is this one ready to merge?

**Comment by kimsauce** [comment]
> @kimsauce, is this one ready to merge?

yes! thank you for the nudge. just merged.

**Pull request merged**